# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linesrv/install.patch.lcc
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lcc/Makefile.vanilla	2006-10-04 16:43:37.000000000 +0000
+++ lcc/Makefile	2006-10-04 16:45:41.000000000 +0000
@@ -6,8 +6,14 @@
 OBJ=lcc.o lcp3.o lcp3_io.o init.o globals.o lists.o user.o proccmd.o end.o
 PRG=lcc
 
+prefix := /usr
+
 all: $(PRG)
 
+install: $(PRG)
+	install $(PRG) $(DESTDIR)$(prefix)/bin/
+	install lcc.conf $(DESTDIR)/etc/
+
 clean:
 	rm *.o
 
